examples-standalone/finance-portfolio/src/app/components/heatmap/heatmap.component.ts

```diff
@@ -14,7 +14,7 @@ declare var kendo: any;
 })
 export class HeatmapComponent implements AfterViewInit, OnDestroy {
 
-    @ViewChild('heatmap') heatmap: ElementRef;
+    @ViewChild('heatmap') heatmap: ElementRef | undefined;
```

Comment:
It's more common to use a nullable field rather than listing `undefined` or `null`:

```ts
@ViewChild('heatmap') heatmap?: ElementRef;
```

(same for other occurrences in this PR)